modules/prompt/init.zsh

```diff
@@ -13,9 +13,12 @@ zstyle -a ':prezto:module:prompt' theme 'prompt_argv'
 if [[ "$TERM" == (dumb|linux|*bsd*) ]] || (( $#prompt_argv < 1 )); then
   prompt 'off'
 elif [[ "$prompt_argv[1]" == 'powerlevel9k' ]] ; then
-  printf "'powerlevel9k' has been deprecated and unsupported by its author, "
-  printf "consider migrating to 'powerlevel10k' instead.\n"
-  printf "Switching to prezto default prompt...\n"
+  <<EOW
+WARNING: Prezto does not support 'powerlevel9k' anymore as it has
+         been deprecated and is not supported by its author.
+         Consider migrating to 'powerlevel10k' instead.
```

Comment:
It'd be nice if this included a brief instruction of how to migrate, or where to go... I know many folks (like my wife) setup their prezto install years ago and don't remember the internals and frankly don't enjoy playing with the guts of stuff like this (how strange! 😀 )... so giving them a pointer toward "to switch, see xyz" would be helpful.

Reply:
See #1902, helps?